src/AllTravelers.js

```diff
@@ -34,7 +34,7 @@ class AllTravelers {
   }
 
   calculateIndividualCost() {
-    return this.trips.filter(trip => trip.status === 'approved').map(trip => {
+    return this.trips.filter(trip => trip.date <= moment().format('YYYY/MM/DD') && trip.status === 'approved').map(trip => {
       const tripsObj = {};
```

Comment:
Will told me that we shouldn't include the dataType in the variable name, so we should call tripsObj something like tripsDetails

Reply:
I like not converting the trip.date into a moment, I converted both to moments but now I see I didn't have to!

Reply:
Oh I do remember him saying that..thanks for the reminder!

Reply:
Really minor nitpick but whenever I see a conditional with an `&&` I think it could time for a refactor.  This isn't the best example but what if we refactored this with a helper method for the conditional.

```
    tripOverAndApproved?(trip) {
      return trip.date <= moment().format('YYYY/MM/DD') && trip.status === 'approved'
    }
```

Then in this function here we can use our helper function

`return this.trips.filter(trip => tripOverAndApproved(trip)).map.........`

Thoughts?

Reply:
Also the function is called `calculateIndividualCost` which seems a bit misleading? Can you think of a better function name?

Reply:
Overall really good job on this method though and using momemt js well!

Reply:
Oh interesting..I never thought of having a helper function for it! And I can see how that name could be misleading..maybe something like `calculateTravelerCost`?